internal/stack/boot.go

```diff
@@ -149,17 +142,16 @@ func dockerComposeDown() error {
 		return errors.Wrap(err, "locating stack directory failed")
 	}
 
-	cmd := exec.Command("docker-compose",
-		"-f", filepath.Join(stackDir, "snapshot.yml"),
-		"--project-directory", stackDir,
-		"down")
+	c, err := compose.NewProject(dockerComposeProjectName, filepath.Join(stackDir, "snapshot.yml"))
+	if err != nil {
+		return errors.Wrap(err, "could not create docker compose project")
+	}
+
 	// We set the STACK_VERSION env var here to avoid showing a warning to the user about
 	// it not being set.
-	cmd.Env = append(os.Environ(), fmt.Sprintf("STACK_VERSION=%s", DefaultVersion))
-	cmd.Stderr = os.Stderr
-	cmd.Stdout = os.Stdout
-	err = cmd.Run()
-	if err != nil {
+	env := []string{fmt.Sprintf("STACK_VERSION=%s", DefaultVersion)}
+
+	if err := c.Down(nil, env); err != nil {
```

Comment:
Same here, you can get rid of passing the "nil" value.

Reply:
Fixed in 48f0a53.